dictdiffer/__init__.py

```diff
@@ -25,9 +25,9 @@
 
 __all__ = ('diff', 'patch', 'swap', 'revert', 'dot_lookup', '__version__')
 
-DICT_TYPES = (MutableMapping, )
-LIST_TYPES = (MutableSequence, )
-SET_TYPES = (MutableSet, )
+DICT_TYPES = (MutableMapping,)
```

Comment:
@yoyonel can you please move the whitespace changes to separate PR with an automatic style check (e.g. YAPF)?

Reply:
Yep, no pb :)